framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/wwwroot/libs/abp/aspnetcore-mvc-ui-theme-shared/datatables/datatables-extensions.js

```diff
@@ -299,6 +299,11 @@
                     column.targets = i;
                 }
 
+                if (!column.render && column.dataFormat){
+                    var render = datatables.defaultRenderers[column.dataFormat];
+                    column.render = render ? render : ISOStringToDateTimeLocaleString(column.dataFormat);
```

Comment:
Do nothing if there is no renderer registered with the given name. We don't assume that the data is always date/datetime. `datatables.defaultRenderers` can have any type of renderer, just like the `datatables.defaultRenderers['boolean']` registered before. 

If we also want to accept luxon-specific format arguments, we can introduce another option like `dataFormatArgs` (optional) which can be an object and passed to the renderer function. But the `dataFormat` always should be simple string that is matched an item in the datatables.defaultRenderers.